common/src/main/java/tachyon/Constants.java

```diff
@@ -139,8 +142,14 @@
   public static final String MASTER_RETRY_COUNT = "tachyon.master.retry";
 
   public static final String WORKER_MEMORY_SIZE = "tachyon.worker.memory.size";
+  public static final String WORKER_HOSTNAME = "tachyon.worker.hostname";
+  public static final String WORKER_BIND_HOST = "tachyon.worker.bind.host";
   public static final String WORKER_PORT = "tachyon.worker.port";
+  public static final String WORKER_DATA_HOSTNAME = "tachyon.worker.data.hostname";
+  public static final String WORKER_DATA_BIND_HOST = "tachyon.worker.data.bind.host";
   public static final String WORKER_DATA_PORT = "tachyon.worker.data.port";
+  public static final String WORKER_WEB_HOSTNAME = "tachyon.worker.web.hostname";
+  public static final String WORKER_WEB_BIND_HOST = "tachyon.worker.web.bind.host";
```

Comment:
There were lots of discussion about the motivation of separating each of services (Spark, HDFS, YARN, MapReduce). If we can let user specify the bind and connect addresses for each of services, it'll be much easier to integrate with other systems. Maybe we could open a best practices page about Tachyon multihomed deployment. If user don't want to scale the services over different networks, at least they can just keep default setting. In the default setting, all services only use homogeneous network.